### ISSUE TITLE
Silent Failure on Model.update()

### ISSUE THREAD

**Issue by ryanquinn3** [balderdashy/waterline#1249]
Version 0.10.17 

I incorrectly was using the update api by just passing in the json of the object to update (which had an id key) and not the find criteria and the call to update did not return. It didn't reject or anything just hung. I know I was incorrectly using the call but I would imagine that it would reject or something if the inputs were not proper.

**Comment by devinivy** [comment]
Can you see if this is an issue in the most recent version of waterline (0.10.28)?

**Comment by sailsbot** [comment]
Thanks for posting, @ryanquinn3.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by prbaron** [comment]
Tried with 0.11 and it is the same. I provided the wrong id and I received a 200. I was expecting an error to be able to return a 404.

**Comment by irlnathan** [comment]
@prbaron Howdy, please open a new issue.  Thanks!

Best, irl
